students/joshua-bone/lesson3/mailroom.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+# Joshua Bone - UW Python 210 - Lesson 3
+# 12/25/2018
+# Assignment: Mailroom, Part 1
+
+menu = ("Welcome to the Mailroom!\n\n\n" +
+        "Please select from the following options:\n" +
+        "(S)end a Thank You\n" +
+        "(C)reate a Report\n" +
+        "(Q)uit\n\n\n")
+donors = {}
+NUM_WIDTH = 13
+TY = ("\n\nDear {:s},\n\nThank you for your generous donation of ${:.2f}.\n\n")
+
+
+def add(name, amt):
+    assert isinstance(amt, (int, float))
+    assert amt > 0
+    assert isinstance(name, str)
+    if name not in donors:
+        donors[name] = []
+    donors[name].append(amt)
```

Comment:
Per instructions:
> use only functions and the basic Python data types you’ve learned about so fa

In this lesson you should have been using a `list` for your main data structure.